test/util.ts

```diff
@@ -0,0 +1,119 @@
+import {Program} from "../src/runtime/dsl";
+import * as Runtime from "../src/runtime/runtime";
+import * as test from "tape";
+
+// You can specify changes as either [e,a,v] or [e,a,v,round,count];
+export type EAVArray = [Runtime.RawValue, Runtime.RawValue, Runtime.RawValue];
```

Comment:
Personal preference, but I think it's clearer to refer to arrays with a fixed set of fields as tuples, since it's a lot more obvious that `myTuple.push(...)` is funky compared to `myArray.push(...)`